### ISSUE TITLE
Locator js for Cursor IDE, on unbuntu

### ISSUE THREAD

**Issue by jawale204** [infi-pc/locatorjs#170]
I am trying to set cursor as my IDE through locatorjs project settings. Attached the SS
![Screenshot from 2024-12-09 12-59-33](https://github.com/user-attachments/assets/6d50c783-af33-472f-9127-10db32b551b3)

When i click on any component the pop up appears but it does not take me anywhere.
![image](https://github.com/user-attachments/assets/c845c459-8039-4ea4-b47f-bed1da1968e8)

**Comment by louisjuliendo** [comment]
There is an invalid placement of `/` in your URL:
- `cursor://file$/{projectPath}${filePath}:${line}:${column}` ❌
- `cursor://file/${projectPath}${filePath}:${line}:${column}` ✔

However, it doesn't automatically open Cursor for unknown reason (works in VSCode tho)

**Comment by madelahsan** [comment]
> There is an invalid placement of `/` in your URL:
> 
> * `cursor://file$/{projectPath}${filePath}:${line}:${column}` ❌
> * `cursor://file/${projectPath}${filePath}:${line}:${column}` ✔
> 
> However, it doesn't automatically open Cursor for unknown reason (works in VSCode tho)

Yeah i am on Linux fedora 41 and Its not working for Cursor. Although Working fine for VSCode
